ingress/controllers/nginx/README.md

```diff
@@ -149,6 +150,23 @@ By default the controller redirects (301) to HTTPS if there is a TLS Ingress rul
 To disable this behavior use `hsts=false` in the NGINX config map.
 
 
+### Automated Certificate Management with Kube-Lego
+
+[Kube-Lego] automatically requests missing certificates or expired from
+[Let's Encrypt] by monitoring ingress resources and its referenced secrets. To
+enable this for an ingress resource you have to add an annotation:
+
+```
+kubectl annotate ing ingress-demo kubernetes.io/tls-acme="true"
```

Comment:
> The way it's currently written, I believe kube-lego will create an ingress for itself that exposes /well-known (just by looking at code, not tried). 
This is correct.

>  On cloudproviders, each ingress is a different public vip,
I might be missing something here, but this is definitely not the case on AWS. I simply have an ELB pointing at all the nodes in the cluster (selecting the nodes running nginx-ingress through healthchecks).